### PR TITLE
New Units of Information Cheat Sheet

### PR DESCRIPTION
**Pull request by SvetlanaZem**
**What does your Instant Answer do?**
Shows information about size of standard units of information, e.g. KB, GB, etc.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It allows you to quickly have a look at data hierarchy and relative/absolutely sizes. 

**What is the data source for your Instant Answer? (Provide a link if possible)**
Wikipedia: https://en.wikipedia.org/wiki/Units_of_information

**Why did you choose this data source?**
It's worldwide use source, it's especially good for encyclopedic knowledge.

**Are there any other alternative (better) data sources?**
There are standard articles (e.g. IEC 80000-13:2008), but it's not freely distributed. 

**What are some example queries that trigger this Instant Answer?**
"units of data cheat sheet", "data size cheat sheet"

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Anyone related to IT.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No.

**Which existing Instant Answers will this one supercede/overlap with?**
No one.

**Are you having any problems? Do you need our help with anything?**
No.

**Where did you hear about DuckDuckHack? (For first time contributors)**
I was wondering if DuckDuckGo is open source or has open source parts, therefore I found it intentionally.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot from 2015-09-26 18 59 37

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Ubuntu 14.04
      - [X] Google Chrome
```

-----
IA Page: https://duck.co/ia/view/digital_information_units_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/data-size.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            "key": "Zettabyte (ZB)"
+        }, {
+            "val": "10^24 bytes = 1000^8 bytes = 1000 ZB",
+            "key": "Yottabyte (YB)"
+        }
+        ],
+        "Binary": [
+        {
+            "val": "2^10 bytes = 1024 bytes",
+            "key": "Kilobyte (KB) / Kibibyte (KiB)"
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/data-size.json]: I think it probably makes most sense to not refer to the binary values (2^n) as SI prefix (kilo/mega) etc at all. It's also then consistent with the conversions: https://duckduckgo.com/?q=1024+bytes+in+kib&t=ffab&ia=answer

**Comment by SvetlanaZem** [comment]
I have removed additional naming for binary data.

File: share/goodie/cheat_sheets/json/data-size.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
+    "id":"data_size_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/data-size.json]: I think something more descriptive like "digital_information_units_cheat_sheet" would be better? I think "data" is a little vague

File: share/goodie/cheat_sheets/json/data-size.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    "name":"Units of Data",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/data-size.json]: Likewise, I think this should be a little more specific?

**Comment by SvetlanaZem** [comment]
I've changed cheat sheet id, title and file name according to your recommendations.

**Comment by mintsoft** [comment]
@moollaza this lgtm?

**Comment by moollaza** [comment]
@SvetlanaZem @mintsoft this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by SvetlanaZem** [comment]
@moollaza Thank you, I would be happy to help if someone needs it.

**Comment by moollaza** [comment]
@SvetlanaZem congrats! It's live: https://duckduckgo.com/?q=digital+information+units+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**